packages/perseus/src/widgets/interactive-graphs/graphs/utils.ts

```diff
@@ -46,8 +46,25 @@ export const getIntersectionOfRayWithBox = (
     }
 };
 
-export const getLines = (points: readonly vec.Vector2[]): CollinearTuple[] =>
-    points.map((point, i) => {
+export const getLines = (points: readonly vec.Vector2[]): CollinearTuple[] => {
+    return points.map((point, i) => {
         const next = points[(i + 1) % points.length];
         return [point, next];
     });
+};
+
+/**
+ * determine radius of a circle given the center point
+ * and a point on the circle's circumference
+ *
+ * @param center - center point of the circle
+ * @param edgePoint - point on the circumference of the circle
+ * @returns the radius of the circle
+ */
+export function getRadius(center: vec.Vector2, edgePoint: vec.Vector2): number {
+    const [centerX, centerY] = center;
+    const [edgeX, edgeY] = edgePoint;
+    return Math.sqrt(
+        Math.pow(edgeX - centerX, 2) + Math.pow(edgeY - centerY, 2),
+    );
+}
```

Comment:
Minor: I think this `getRadius` function could live with the definition of the circle graph state type, and take the entire graph state as its argument:

```ts
getRadius(circleGraph);
```

...and then `getRadius` could live in `interactive-graph-state.ts`.

Our way of representing a circle, as a center point and a point on the edge, is pretty specific to the graph state format. Passing the entire graph state to `getRadius` would better encapsulate the structure of the state, and possibly let us change it more easily in the future.

(in general, the practice of passing entire state objects around opens up other possibilities, too, like caching the results of getter functions — see [`flipbook-model.ts`](https://github.com/Khan/perseus/blob/db041577dda780e2a871ad9e63c295a1d3fbe185/dev/flipbook-model.ts#L123-L152) for examples. I don't think caching is relevant here, though)